setup.py

```diff
@@ -58,8 +58,10 @@ def run(self):
 
         if self.type in ['unit', 'all']:
             cmd.append('python/Ganga/new_tests/Unit')
-            cmd.append('python/Ganga/Core')
-            cmd.append('python/Ganga/Runtime')
+            ## Ganga/Core finds zero tests
+            #cmd.append('python/Ganga/Core')
+            ## Ganga/Runtime imports GangaProgram but causes a vanilla instance to be loaded
+            #cmd.append('python/Ganga/Runtime')
```

Comment:
There is  default config being created by the Ganga/Runtime line which keeps asking for a grid proxy to be created if one doesn't exist.
The LHCb.ini should cause this not to happen but loading of Ganga/Runtime causes the the Remote code to be loaded which asks for a grid password.